docs/SequenceDiagrams/EditSeqDiagrams/GetModule.puml

```diff
@@ -0,0 +1,34 @@
+@startuml
+'https://plantuml.com/sequence-diagram
+
+skinparam shadowing false
+participant ":EditCommand" as EditCommand
+participant ":ModuleList" as ModuleList
+hide footbox
+
+mainframe **sd** Get Module
+
+activate EditCommand
+EditCommand -> EditCommand: getTargetModule(moduleList)
+activate EditCommand
+alt Objects.isNull(taskModule)
+    EditCommand -> ModuleList: getGeneralTasks()
+    activate ModuleList
+    return
+
+else else
+    EditCommand -> ModuleList: getModule(taskModule)
+    activate ModuleList
+
+    alt Objects.isNull(taskModule)
+        [<-- ModuleList: throw NoSuchModuleException
+
+    else else
+    return targetModule
+
+    end
+end
+return
+deactivate EditCommand
+
+@enduml
```

Comment:
Perhaps simplifying this diagram (there are 2 alt frames with the same condition and not sure the else condition belongs to which alt frame) and making it clearer would help to aid understanding